apps/changelog/src/client/components/date.tsx

```diff
@@ -1,11 +1,10 @@
 const formatDate = (date: string | Date) => {
-  const options: Intl.DateTimeFormatOptions = {
+  const now = new Date(date).toLocaleDateString('en-EN', {
     year: 'numeric',
     month: 'long',
     day: 'numeric',
     timeZone: 'UTC'
-  };
-  const now = new Date(date).toLocaleDateString('en-EN', options);
+  });
```

Comment:
l: we could reuse this formatting, or maybe just use day.js?

Reply:
I think this is slimmer for now!